scripts/sct_dmri_compute_bvalue.py

```diff
@@ -47,31 +46,44 @@ def main():
     sct.printv('  gyromagnetic ratio ..... ' + str(GYRO) + ' Hz/T')
     sct.printv('')
 
-    bvalue = (2 * math.pi * GYRO * gradamp * 0.001 * smalldelta * 0.001) ** 2 * (bigdelta * 0.001 - smalldelta * 0.001 / 3)
+    bvalue = (2 * math.pi * GYRO * gradamp * 0.001 * smalldelta * 0.001) ** 2 * (
+    bigdelta * 0.001 - smalldelta * 0.001 / 3)
 
-    sct.printv('b-value = ' + str(bvalue / 10**6) + ' mm^2/s\n')
+    sct.printv('b-value = ' + str(bvalue / 10 ** 6) + ' mm^2/s\n')
     return bvalue
 
 
 def get_parser():
     # Initialize the parser
-    parser = Parser(__file__)
-    parser.usage.set_description('Calculate b-value (in mm^2/s).')
-    parser.add_option(name="-g",
-                      type_value="float",
-                      description="Amplitude of diffusion gradients (in mT/m)",
-                      mandatory=True,
-                      example='40')
-    parser.add_option(name="-b",
-                      type_value="float",
-                      description="Big delta: time between both diffusion gradients (in ms)",
-                      mandatory=True,
-                      example='40')
-    parser.add_option(name="-d",
-                      type_value="float",
-                      description="Small delta: duration of each diffusion gradient (in ms)",
-                      mandatory=True,
-                      example='30')
+    parser = argparse.ArgumentParser(
+        description='Calculate b-value (in mm^2/s).',
+        add_help = None,
+        prog=os.path.basename(__file__).strip(".py"))
+    mandatory = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatory.add_argument(
+        "-g",
+        type = float,
+        metavar = Metavar.float,
+        help="Amplitude of diffusion gradients (in mT/m) (e.g.,'40')",
```

Comment:
I prefer [this format](https://github.com/neuropoly/spinalcordtoolbox/pull/2315/files#diff-f695801b7691edf1495001cad74eb71aR85) for examples. I find it a bit less heavy and also less confusing. I imagine some people will enter '40' instead of 40. Also, some people are not familiar with the acronym e.g.

Reply:
Sounds good! Changes made in following functions:

sct_deepseg_gm: Commit [6b8c267](https://github.com/neuropoly/spinalcordtoolbox/commit/6b8c26724798fa69087a4cf1062ac8a36c60e682)
sct_deepseg_lesion: Commit [13e9a6d](https://github.com/neuropoly/spinalcordtoolbox/commit/13e9a6d6fcb3e2fe8bc34f7cc89ccdc612b6f990)
sct_deepseg_sc: Commit [5a140a2](https://github.com/neuropoly/spinalcordtoolbox/commit/5a140a290546ea3e7c6b207edc6486312641ab7f)
sct_denoising_onlm: Commit [d726df9](https://github.com/neuropoly/spinalcordtoolbox/commit/d726df9cc8b552c1deaedf71d2659233a30ec829)
sct_detect_pmj: Commit [c842bb4](https://github.com/neuropoly/spinalcordtoolbox/commit/c842bb4e6a0c5893bae38d7439eaa2d67dd5167a)
sct_dice_coefficient: Commit [8a9e991](https://github.com/neuropoly/spinalcordtoolbox/commit/8a9e991c988532b6840a7b5adea16beab8771944)
sct_dmri_compute_bvalue: Commit [f353c37](https://github.com/neuropoly/spinalcordtoolbox/commit/f353c373c7afd331fac1fb3f05a0291ff3a4351e)
sct_dmri_compute_dti: Commit [4f07d68](https://github.com/neuropoly/spinalcordtoolbox/commit/4f07d68f3a7c9a943b2dd453d15e481d7a2d6204)
sct_dmri_concat_bvals: Commit [1f54afb](https://github.com/neuropoly/spinalcordtoolbox/commit/1f54afbde4bf420168cf621435d8c814ae370418)
sct_dmri_concat_bvecs: Commit [4e30952](https://github.com/neuropoly/spinalcordtoolbox/commit/4e3095267fb23a7e0b198d2fac70bb75d6b6a009)